src/mmw/sass/components/_modals.scss

```diff
@@ -138,3 +138,20 @@
   top: 1rem;
   left: 2rem;
 }
+
+.fa {
+  font-size: 1.0rem;
+  padding: 0 0.1rem 0 0.1rem;
+}
```

Comment:
These changes are scoped broadly enough to impact some of the other app icons:

![screen shot 2016-10-05 at 2 37 34 pm](https://cloud.githubusercontent.com/assets/4165523/19126554/634b1a9c-8b09-11e6-99ee-182fb33ab825.png)